decidim/decidim
# Redesign: fix emoji popup position

#### :tophat: What? Why?

This PR fixes the emoji popup position by just upgrading the package version, because it was an error in the library https://github.com/joeattardi/picmo/issues/259

#### :pushpin: Related Issues

- Fixes https://github.com/decidim/decidim/issues/10627

#### Testing

1. Login
2. Visit any comments thread
3. Click on the emojis button and check the openning popup is centered

### :camera: Screenshots

![Screenshot 2023-06-06 at 11 25 03](https://github.com/decidim/decidim/assets/17616/1184130d-09d0-4e8c-89ae-52a598a6573e)

:hearts: Thank you!
